plugin/src/main/java/spp/jetbrains/sourcemarker/Constants.java

```diff
@@ -0,0 +1,14 @@
+package spp.jetbrains.sourcemarker;
+
+import java.util.regex.Pattern;
+
+public interface Constants {
```

Comment:
This class feels superfluous. Everything shared between classes should have moved to VariableParser. Things like `WAITING_FOR_LIVE_LOG_DATA` and `SPACE` aren't used anywhere but LogStatusBar. They have no point in being in a shared place.

Reply:
Removed and constants moved to classes wherever its primarily used